### ISSUE TITLE
Support underscore templates

### ISSUE THREAD

**Issue by donaldpipowitch** [beautify-web/js-beautify#1130]
# Description

Taken from https://github.com/Microsoft/vscode/issues/20197.

# Motivation

Underscore templates within HTML are a common feature in solutions like [html-webpack-plugin](https://github.com/jantimon/html-webpack-plugin).

# Input

The code looked like this before beautification:

```html
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <% console.log('hello') %>
  <title>Login</title>
</head>
<body>
  <%
    // copy configs
    require('copy-loader!./foo.js');
  %>
  <script src="foo.js"></script>
</body>
</html>
```

# Expected Output

The code should have looked like this after beautification: _no change_.

# Actual Output

The  code actually looked like this after beautification:

```html
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <% console.log('hello') %>
    <title>Login</title>
</head>
<body>
  <%
    // copy configs
    require('copy-loader!./foo.js');
  %>
    <script src="foo.js"></script>
</body>
</html>
```

It looks like every tag after `<% %>` is indented by two spaces.

# Steps to Reproduce

* VS Code Version: Version 1.10.0-insider (1.10.0-insider)
* VS Code Setting: `"html.format.enable"` is `true`.

## Environment

* OS Version: Mac OS X 10.12.3

**Comment by bitwiseman** [comment]
So, there's an over indent in the element following the template.   Got it.

**Comment by CNSKnight** [comment]
I'm trying a workaround for this quirk:
```
// in your ../User/.jsbeautifyrc
{
    "html":
    {
        "unformatted": ["%", "a", "span", "img", "code", "pre", "sub", "sup", "em", "strong", "b", "i", "u", "strike", "big", "small", "pre", "h1", "h2", "h3", "h4", "h5", "h6"]

    }
}
```

**Comment by victornpb** [comment]
adding `%` to unformatted tags does't work for me

**Comment by divyamrastogi** [comment]
Is this possible? Having beautify do this would save me and a lot of other developers working on BackboneJS from a lot of pain.

**Comment by MacKLess** [comment]
This appears to have been resolved as of v1.8.0-rc13. @donaldpipowitch and @CNSKnight please double check and let me know if you disagree.
